src/quacc/recipes/orca/core.py

```diff
@@ -81,6 +82,78 @@ def static_job(
     )
 
 
+@job
+def freq_job(
+    atoms: Atoms,
+    charge: int = 0,
+    spin_multiplicity: int = 1,
+    xc: str = "wb97x-d3bj",
+    basis: str = "def2-tzvp",
+    numeric: bool = False,
```

Comment:
Mind if we change this to `numerical`? Feels more boolean to me. (now I'm thinking of alphabet vs alphabetical in my head...).